CBC.js

```diff
@@ -0,0 +1,436 @@
+{
+	"translatorID": "03c4b906-8cb2-4850-a771-697cbd92c2a1",
+	"label": "CBC",
+	"creator": "Geoff Banh",
+	"target": "^https?:\\/\\/www\\.cbc\\.ca",
+	"minVersion": "5.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 12,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2024-03-08 23:10:27"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2024 Geoff Banh
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, url) {
+	let path = new URL(url).pathname;
+	if (/search/.test(path) && getSearchResults(doc, true)) {
+		return 'multiple';
+	}
+	else if ((/(news|sports|radio|books|arts|music|life|television|archives)\//.test(path)) && getLD(doc)) {
+		return "newspaperArticle";
+	}
+	else if (/player/.test(path)) {
+		return "videoRecording";
+	}
+	return false;
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	// Adjust the CSS Selectors
+	var rows = doc.querySelectorAll('.card.cardListing.rightImage');
```

Comment:
Sure, doesn't look like that level of specificity is required actually, since only entries have `.card.cardListing`.
```suggestion
	var rows = doc.querySelectorAll('.card.cardListing');
```